### DISCUSSION TITLE
Can this code run on raspberry pi model 4

### DISCUSSION THREAD

**Discussion by Ana119** [alibaba/FederatedScope#183] [Q&A] [answered]
Can this code run on raspberry pi model 4? 
![image](https://user-images.githubusercontent.com/106262211/175067141-dcbc5009-ebf6-4ce0-ab4f-32427b58a2ed.png)

**Comment by xieyxclack** [comment, marked as answer]
We have not conducted experiments on raspberry pi, and we would greatly appreciate if you would like to make such a test and contribute to FederatedScope!
